Dockerfile.dev

```diff
@@ -0,0 +1,57 @@
+### Build and install packages
+FROM python:3.6 as build-python
+ARG STATIC_URL
+ARG SKIP_LOCK
+ARG HOST_UID
+ARG HOST_GID
+
+RUN apt-get -y update && \
+    apt-get install -y gettext && \
+    # Cleanup apt cache
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install Python dependencies
+RUN pip install pipenv
+ADD Pipfile /app/
+ADD Pipfile.lock /app/
+WORKDIR /app
+RUN pipenv install $SKIP_LOCK --system --deploy --dev
+
+
+### Final image
+FROM python:3.6-slim
+ARG STATIC_URL
```

Comment:
This is not really 100% part of this PR but as I see more ENV and ARG variables added, the disorganization grows.

Would you mind putting all the variables together like so:

```
ARG STATIC_URL
ARG SKIP_LOCK
ARG HOST_UID
ARG HOST_GID
ENV STATIC_URL=${STATIC_URL:-/static/} \
  HOST_UID=${HOST_UID:-1000} \
  HOST_GID=${HOST_GID:-HOST_UID} \
  PORT=8000 \
  PYTHONUNBUFFERED=1 \
  PROCESSES 4
```